.devcontainer/devcontainer.json

```diff
@@ -70,5 +70,5 @@
   "postCreateCommand": "echo 'source /usr/share/bash-completion/completions/git' >> ~/.bashrc",
 
   // Use 'forwardPorts' to make a list of ports inside the container available locally.
-  "forwardPorts": [9000, 5432, 3390]
+  "forwardPorts": [9000, 5432, 3390, 9999]
```

Comment:
(non-blocking) sort these please

Reply:
Updated!

Reply:
I don't know if we need all of these
localstack: 4566, 6645, needed for the check email script or checking file uploads 
azureite: 10000, never needed it myself
civiform in unit tests: 9100, but I've never used that one
prod simulator: 8888, unlikely to be run on local machine, but never know